lib/wings/orm_converter.rb

```diff
@@ -62,7 +63,13 @@ class << self
           attr_reader :internal_resource
 
           def name
-            ancestors[1..-1].find { |parent| parent < ::Valkyrie::Resource }&.name
+            _canonical_valkyrie_model&.name
+          end
+
+          ##
+          # @api private
+          def _canonical_valkyrie_model
```

Comment:
Hmm. Does `@api private` reconcile with the above usage in lookup? I'm thinking it might, but want to double check.

Reply:
i think yes. this tag is meant to communicate to applications that this isn't stable/public API. in this case, i think it's `Wings`-use only.